ArduCopter/mode_autorotate.cpp

```diff
@@ -82,28 +84,87 @@ void ModeAutorotate::run()
         set_mode(copter.prev_control_mode, ModeReason::AUTOROTATION_BAILOUT);
     }
 
+    float pilot_roll, pilot_pitch, pilot_yaw_rate;
+    if (g2.arot._param_guided == 1){
```

Comment:
Should also have some sort of timeout on the command.

Reply:
Guided mode control was removed.